src/io/flutter/run/FlutterDebugProcess.java

```diff
@@ -53,6 +53,11 @@ public FlutterDebugProcess(@NotNull FlutterApp app,
     this.app = app;
   }
 
+  @NotNull
```

Comment:
btw when we create the InspectorService we probably have an app available. We could just flow the dependency in that way.

Reply:
Good point.  I need to do some reconsidering when we do more source navigation stuff.  Will revisit.